docker-compose-develop.yml

```diff
@@ -3,12 +3,14 @@ services:
   develop:
     build: .
     ports:
-      - "35749:35729"
+      - "5001:5001"
     environment:
+      PORT: 5001
+      NODE_PATH: app/src
       CT_URL: http://mymachine:9000
-      LOCAL_URL: http://mymachine:5000
+      LOCAL_URL: http://mymachine:5001
       API_VERSION: v1
-      CT_TOKEN: null
+      CT_TOKEN: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Im1pY3Jvc2VydmljZSIsImNyZWF0ZWRBdCI6IjIwMTYtMDktMTQifQ.IRCIRm1nfIQTfda_Wb6Pg-341zhV8soAgzw7dd5HxxQ
```

Comment:
@samdavidge same question as https://github.com/gfw-api/gfw-prodes-loss-api/pull/49

Reply:
Hey @ethanroday this is a local/test key that is used (and committed) across many of the repos.